docs-next/pages/guides/configuring-relationships.mdx

```diff
@@ -0,0 +1,243 @@
+
+### One-sided
+
+If we want to know who the author of a post is, but we're not interested in querying for all the posts written by a given user we can set up a one-sided relationship as follow:
+
+```typescript
+import { config, createSchema, list } from '@keystone-next/keystone/schema';
+import { text, relationship } from '@keystone-next/fields';
+
+export default config({
+  lists: createSchema({
+    User: list({ fields: { name: text() } }),
+    Post: list({
+      fields: {
+        title: text(),
+        content: text(),
+        author: relationship({ ref: 'User', many: false }),
+      }
+    })
+  }),
+});
+```
+
+In the `author` field we have specified `ref: 'User'` to indicate that this field relates to an item in the `User` list.
+We can now write the following query to find the author for each post:
+
+```graphql
+Query {
+  allPosts {
+    title
+    content
+    author {
+      name
+    }
+  }
+}
+```
+
+### Two-sided
+
+If we also want to access all the posts written by a user then we need to use a two-sided relationship.
+
+```typescript
+export default config({
+  lists: createSchema({
+    User: list({
+      fields: {
+        name: text(),
+        posts: relationship({ref: 'Post.author', many: true }),
+      }
+    }),
+    Post: list({
+      fields: {
+        title: text(),
+        content: text(),
+        author: relationship({ ref: 'User.posts', many: false }),
+      }
+    })
+  }),
+});
+```
+
+In this case we have a `relationship` field on both lists, and they both have a `ref` config in the form `<listName>.<fieldName>`.
+These `relationship` fields represent the two sides of the relationship, and we can now use the following GraphQL query as well:
+
+```graphql
+Query {
+  allUsers {
+    name
+    posts {
+      title
+      content
+    }
+  }
+}
+```
+
+## Cardinality
+
+The second question we need to ask is what the _cardinality_ of our relationship should be.
+The _cardinality_ of a relationship is the number items which can exist on either side of the relationship.
+In our blog do we want each post to have exactly one author, or can it have multiple authors?
+Are users allowed to write more than one post or do we want to restrict them to exactly one post each for some reason?
+The answers to these questions will give us the cardinality of our relationship.
+
+There are three types of cardinality, `one-to-many`, `many-to-many`, and `one-to-one`, and they can be configured using the `many` config option.
+
+### One-to-many
+
+If we want a blog where each post can have **one** author, and each user can be the author of **many** posts, then we have a `one-to-many` relationship.
+We can configure a one-sided version of this:
+
+```typescript
+import { config, createSchema, list } from '@keystone-next/keystone/schema';
+import { text, relationship } from '@keystone-next/fields';
+
+export default config({
+  lists: createSchema({
+    User: list({
+      fields: {
+        name: text()
```

Comment:
This one doesn't have a trailing comma while the other bits do... not a blocker... just pointing out.